### DISCUSSION TITLE
yolov8-pose

### DISCUSSION THREAD

**Discussion by creeperstudio1** [ultralytics/ultralytics#12586] [Q&A]
Is there a way to only show and calculate certain keypoints of the body. For example, I only want the model to show and calculate por the head, or just for lower body, if so, where can I activate or deactivate these features.

**Comment by pderrenger** [comment]
Hello!

Yes, you can customize which keypoints are calculated and displayed by modifying the keypoints list in your inference script. When you receive the keypoints from the model, you'll typically get them for all body parts detected. Here’s a basic example using Python:

```python
from ultralytics import YOLO

# Load your YOLOv8 pose model
model = YOLO('yolov8n-pose.pt')

# Perform inference
results = model('input_image.jpg')

# Process keypoints for specific parts (e.g., head)
for person in results.keypoints:
    head_keypoints = person[:, :5]  # Assuming first 5 keypoints correspond to the head
    print(head_keypoints)  # Do something with the head keypoints
```

You should also check the model documentation for the specific ordering of keypoints to correctly filter out those that pertain to parts of interest, such as the head or lower body.

Hope this helps! Let me know if you need more guidance. 😊
